package.json

```diff
@@ -12,7 +13,7 @@
     "gatsby-link": "^1.6.40",
     "gatsby-plugin-react-helmet": "^2.0.10",
     "gatsby-plugin-react-svg": "^1.1.1",
-    "intersection-observer": "~0.5.0",
+    "intersection-observer": "^0.5.0",
```

Comment:
We should keep the `~` for packages under `1.0.0`.

Reply:
Sure! And thanks for remembering because I have also to remove the old intersection observer.